ui/admin/app/controllers/scopes/scope/workers/worker/tags.js

```diff
@@ -44,6 +47,34 @@ export default class ScopesScopeWorkersWorkerTagsController extends Controller {
     );
   }
 
+  updateApiTags() {
```

Comment:
Not blocking comment: do we plan to add test for this method?

Reply:
This method is used by `editApiTag` which does have tests so I didn't feel it needed. However, I can add some for this as well.

Reply:
Tests added.